lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const mediaParser = require("postcss-media-query-parser").default;
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+const valueParser = require("postcss-value-parser");
+
+const ruleName = "no-duplicate-at-import-rules";
+
+const messages = ruleMessages(ruleName, {
+  rejected: atImport => `Unexpected duplicate @import rule ${atImport}`
+});
+
+const rule = function(actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual });
+    if (!validOptions) {
+      return;
+    }
+
+    const imports = {};
+
+    root.walkAtRules("import", atRule => {
```

Comment:
`@import` case insensitivity use `/^import$/i`, also need more tests around `@IMPORT`, `@ImPoRt` and etc